PowerPointLabs/PowerPointLabs/ActionFramework/TooltipsLab/CreateCallout/CreateCalloutActionHandler.cs

```diff
@@ -17,27 +18,19 @@ class CreateCalloutActionHandler : ActionHandler
         protected override void ExecuteAction(string ribbonId)
         {
             this.StartNewUndoEntry();
-
             PowerPointSlide currentSlide = this.GetCurrentSlide();
-
-            if (currentSlide == null)
-            {
-                return;
-            }
-
             PowerPoint.Selection selection = this.GetCurrentSelection();
 
-            if (selection.Type != PowerPoint.PpSelectionType.ppSelectionShapes)
+            if (currentSlide == null || !ShapeUtil.IsSelectionShape(selection))
             {
-                MessageBox.Show(TooltipsLabText.ErrorNoTriggerShapeSelected);
```

Comment:
Is there a reason for removing this error message?

Reply:
Hi @YuPeiHenry the reason for removing this error message is because the buttons will no longer be enabled when there is no active shape selection.